servers/audio/effects/audio_effect_compressor.h

```diff
@@ -44,7 +44,7 @@ class AudioEffectCompressorInstance : public AudioEffectInstance {
 	int current_channel;
 
 public:
-	void set_current_channel(int p_channel) { current_channel = p_channel; }
+	void set_current_channel(int p_channel) override { current_channel = p_channel; }
```

Comment:
```suggestion
	virtual void set_current_channel(int p_channel) override { current_channel = p_channel; }
```
Same for format, both must be used